sceptre/template.py

```diff
@@ -234,6 +234,19 @@ def _bucket_region(self, bucket_name):
     def _domain_from_region(region):
         return "com.cn" if region.startswith("cn-") else "com"
 
+    @staticmethod
+    def _iterate_entry_points(group, name):
+        """
+        Helper to determine whether to use pkg_resources or importlib.metadata.
+        https://docs.python.org/3/library/importlib.metadata.html
+        """
+        if sys.version_info < (3, 10):
+            from pkg_resources import iter_entry_points
+            return iter_entry_points(group, name)
+        else:
+            from importlib.metadata import entry_points
+            return entry_points(group=group, name=name)
+
```

Comment:
The duplication here still makes me a bit sad, though I can see how that name parameter makes it a bit different.

Reply:
i'm open to suggestions.